src/style.css

```
@@ -0,0 +1,39 @@
+* {
+  margin: 0;
+}
+
+body {
+  margin: 2px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-repeat: no-repeat;
+  background-image: url(images/star-active.svg);
+  margin-right: 6px;
+}
+
+.stars--0 .stars__star {
+  background-image: url(images/star.svg);
+}
+
+.stars--1 .stars__star:nth-of-type(n+2) {
+  background-image: url(images/star.svg);
+}
+
+.stars--2 .stars__star:nth-of-type(n+3) {
+  background-image: url(images/star.svg);
+}
+
+.stars--3 .stars__star:nth-of-type(n+4) {
+  background-image: url(images/star.svg);
+}
+
+.stars--4 .stars__star:nth-of-type(n+5) {
+  background-image: url(images/star.svg);
+}
```

```suggestion
.stars--0 .stars__star,
.stars--1 .stars__star:nth-of-type(n+2),
.stars--2 .stars__star:nth-of-type(n+3),
.stars--3 .stars__star:nth-of-type(n+4),
.stars--4 .stars__star:nth-of-type(n+5) {
  background-image: url(images/star.svg);
}
```
combine this into one